src/pages/en/guides/styling.md

```diff
@@ -176,6 +176,166 @@ You can also use the `<link>` element to load a stylesheet on the page. This sho
+
+```astro title="MyComponent.astro"
+<style>
+  h1 { color: red }
+  div > h1 {
+    color: purple
+  }
+</style>
+<div>
+  <h1 class="red">
+    This header will be purple!
+  </h1>
+</div>
+```
+
+If two rules have the same specificity, the _order of appearance_ is evaluated, and the last rule will take precedence:
+```astro title="MyComponent.astro"
+<style>
+  h1 { color: purple }
+  h1 { color: red }
+</style>
+<div>
+  <h1 class="red">
+    This header will be red!
+  </h1>
+</div>
+```
+
+Among Astro features, if the specificity of a rule is the same, the resulting styles will appear in this order:
+
+- **`<link>` tags in the head** (lowest precedence)
+- **imported styles**
+- **scoped styles** (highest precedence)
+
+### Scoped Styles 
+
+Using [scoped styles](#scoped-styles) does not increase the _specificity_ of your CSS, but it will always come last in the _order of appearance_. They will therefore take precedence over other styles of the same specificity. For example, if you import a stylesheet with the same rule as a scoped style, the scoped style will apply:
+
+```css title="make-it-purple.css"
+h1 {
+  color: purple;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be red!
+  </h1>
+</div>
+```
+
+If you make the imported style _more specific_, it will have higher precedence over the scoped style:
+
+```css title="make-it-purple.css"
+div > h1 {
+  color: purple;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be purple!
+  </h1>
+</div>
+```
+
+### Import Order
+
+When importing multiple stylesheets, if any styles have the same specificity, the _last one imported_ wins:
+
+```css title="make-it-purple.css"
+div > h1 {
+  color: purple;
+}
+```
+```css title="make-it-green.css"
+div > h1 {
+  color: green;
+}
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-green.css"
+import "./make-it-purple.css"
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be purple!
+  </h1>
+</div>
+```
+
+If a component imports CSS, that CSS will be imported at the same time, even if the component is never used:
+```astro title="PurpleParent.astro"
+---
+import "./make-it-purple.css"
+---
+<div>
+  <slot/>
+</div>
+```
+```astro title="MyComponent.astro"
+---
+import "./make-it-green.css"
+import PurpleParent from "./PurpleParent.astro";
+---
+<style>
+  h1 { color: red }
+</style>
+<div>
+  <h1>
+    This header will be purple!
+  </h1>
+</div>
+```
+
+### Link Tags
+During compilation, scoped styles and imported stylesheets are added to the _bottom of the head_ as `<link>` tags, after any other link tags.
+
+This means that if you load a style sheet via a [link tag](#load-a-static-stylesheet-via-link-tags) in the head, it will always have lower precedence:
```

Comment:
And, I do find this a bit confusing too. If this is my `.astro` component as written, there is nothing being added to the head _of my original source component_ that I didn't explicitly write? That's what happens to create the HTML that is sent to the browser, but that doesn't change what's in my actual component file? So, I don't really like that we immediately start talking about turning things into HTML link elements when we really need to tell people when these links as written in their project will be evaluated in comparison to other styling options in their project.

And again, I'd think about rephrasing this in terms of the two factors, specificity and order of appearance, that you've established at the beginning. It's like if we give those rules to follow (Specificity first, then order of appearance), then every explanation only needs speak in those terms. Assuming they know what specificity means, then the real job of this section is to explain Astro's ORDER of evaluating everything. That's what they mostly need to know.

I've rewritten something, which I'm not sure is what you intended, but is what the example seems to want to be explaining.

```suggestion
Style sheets loaded via [link tags](#load-a-static-stylesheet-via-link-tags) in the head are evaluated before any other styles in an Astro component, in the order in which they appear. Therefore, these styles will always have lower precedence than imported styleshees and scoped styles:
```

The other stuff feels more like a "fun fact: ALL styles eventually become stylesheet links in the browser HTML!" if you want to try to include it. But, I don't like it as the description of the *reason* for the styling of this example.

> When Astro bundles and processes your CSS, scoped styles and imported stylesheets are added to the _bottom of the head_ as `<link>` tags, after any other link tags. Your browser then evaluates these link tags in order of their appearance.

Reply:
Gotcha, I can see how referring to the head of the site introduces more confusion than it explains. I took your suggestion and just swapped the clauses to make it slightly more clear, and removed my unnecessary "always":

> Style sheets loaded via [link tags](#load-a-static-stylesheet-via-link-tags) in the head are evaluated in order, before any other styles in an Astro component. Therefore, these styles will have lower precedence than imported stylesheets and scoped styles: